README.md

```
@@ -1,39 +1,246 @@
+sudo darwin-rebuild switch --flake ./nix/darwin
+```
+
+Or use the `just` task runner (see Development section):
+
+```shell
+just switch
+```
+
+## 📁 Project Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)